admin/src/pages/Users.tsx

```diff
@@ -1,184 +1,138 @@
-import { useState } from "react";
-import Navbar from "../components/Navbar"
+import { useState, useEffect } from "react";
+import axios from "axios";
+import Navbar from "../components/Navbar";
 import SideBar from "../components/SideBar";
+import { useRecoilValue } from "recoil";
+import { tokenState } from "../store/atoms/auth";
+import toast from "react-hot-toast";
+
+export interface IUser {
+  id: string;
+  username: string;
+  email: string;
+  verified: boolean;
+  blocked: boolean;
```

Comment:
change the attribute name according to naming convention.

Refer these links: 
1. https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
2. https://www.syncfusion.com/blogs/post/top-javascript-naming-convention

Reply:
We aren't using any naming convention in this project so, this isn't any issue from the beginning!

Reply:
Naming convention comes from language irrespective of project. 

Refer this for Typescript:

https://gist.github.com/anichitiandreea/e1d466022d772ea22db56399a7af576b#naming-booleans

Reply:
Hey @akbatra567, these variable names are coming from Prisma schema and to edit these, there will be a lot of changes in whole applications since these names are used in various places. I think that we can raise a separate issue for changing this throughout since this will be out of the scope of this pull request.

Reply:
Sure then, if you feel that way. Approving this then.